packages/console/package.json

```diff
@@ -69,6 +68,7 @@
     "react-paginate": "^8.1.2",
     "react-router-dom": "^6.2.2",
     "react-syntax-highlighter": "^15.5.0",
+    "recharts": "^2.1.10",
```

Comment:
recharts seems unnecessary in this PR, should remove it to avoid conflict

Reply:
Hey I think there's something wrong in this PR, some of the changes in #1076 are also included in. Please check and rebase

Reply:
PTAL